MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,37 @@
+# Maintainers' guide
+
+## Maintainer list
+
+- Core: https://github.com/containerd/project/blob/main/MAINTAINERS
+- Non-core: [`MAINTAINERS`](./MAINTAINERS)
+
+## Governance
+
+See https://github.com/containerd/project/blob/main/GOVERNANCE.md
+
+## Creating a release
+
+Eligibility to be a release manager:
+- MUST be an active Commiter (Core or Non-core)
+- MUST have the GPG fingerprint listed in [`MAINTAINERS`](./MAINTAINERS)
+- MUST upload the GPG public key to `https://github.com/USERNAME.gpg`
+- MUST protect the GPG key with a passphrase or a hardware token.
+
+Release steps:
+- Open a PR to keep the dependencies up-to-date.
```

Comment:
Does it make sense to expand this to which list of dependencies? Specifically that are external binaries? It can be done in a later PR if it makes sense.

Reply:
Fixed